src/test/java/seedu/internhunter/logic/commands/util/CommandUtilTest.java

```diff
@@ -161,4 +163,19 @@ public void getProfile_validIndex_noExceptionThrown() {
         assertDoesNotThrow(() -> getProfileItem(modelWithData, Index.fromOneBased(1)));
         assertDoesNotThrow(() -> getProfileItem(modelWithData, Index.fromZeroBased(0)));
     }
+
+    @Test
+    public void getFullListIndex_equals_success() {
+        ProfileItem profileItem = profileItemBuilder.withTitle("hello world").withDescriptors("ok").build();
+        modelWithData.addProfileItem(profileItem);
+        ApplicationItem applicationItem = applicationItemBuilder.withInternshipItem(LAZADA_DS)
```

Comment:
I noticed for some fields you use the predefined fields like LAZADA_DS but some you used random strings, maybe stick to using one?

Reply:
fixed